diff --git a/source/auth_credentials.c b/source/auth_credentials.c
index ec6e964..60bd73d 100644
--- a/source/auth_credentials.c
+++ b/source/auth_credentials.c
@@ -12,6 +12,7 @@
 #include <aws/common/string.h>
 #include <aws/http/proxy.h>
 #include <aws/io/tls_channel_handler.h>
+#include <aws/auth/auth.h>
 
 static const char *s_capsule_name_credentials = "aws_credentials";
 static const char *s_capsule_name_credentials_provider = "aws_credentials_provider";
